app/assets/javascripts/discourse/tests/acceptance/admin-watched-words-test.js

```diff
@@ -136,6 +136,41 @@ acceptance("Admin - Watched Words", function (needs) {
     assert.strictEqual(query(".modal-body li .match").innerText, "Hello");
     assert.strictEqual(query(".modal-body li .tag").innerText, "greeting");
   });
+
+  test("emoji replacement", async function (assert) {
```

Comment:
This looks like a copy of the "test modal - replace" test case, doesn't look like we need it.

Reply:
Thanks, I've removed it now 👍🏽